string address
geometry_msgs/Pose[] poses